app/Http/routes.php

```diff
@@ -43,6 +43,17 @@
 Route::get('project/view/{project}', 'ProjectController@view');
 Route::get('project/{project}', 'ProjectController@view'); // overloads go last.
 
+/*--------------------------------*
+ *  Project Comments Controller   *
+ *--------------------------------*/
+Route::post('project/{project}/comments', 'ProjectCommentsController@addComment');
+Route::get('project/comments/edit/{projectComment}', 'ProjectCommentsController@editComment');
```

Comment:
L#50 looks good, but I'm not crazy about these Routes really. The urls don't really seem like they represent the actions that they will be doing.